ui/app/AppLayouts/Chat/panels/InlineSelectorPanel.qml

```diff
@@ -139,6 +141,7 @@ Item {
                                         }
                                     } else {
                                         if (event.key === Qt.Key_Backspace && edit.text === "") {
+                                            suggestionsDialog.forceHide = false
```

Comment:
Why only in that case we reset `forceHide`? What if the user selects by mouse or does "ctrl+A" and starts typing?

Reply:
subscribed on onTextEdited